src/scheduler/parse.c

```diff
@@ -594,13 +594,15 @@ parse_config(char *fname)
 								if (filename == NULL)
 									error = 1;
 								else {
-									int err;
-									err = tmp_file_sec_user(filename, 0, 1, S_IWGRP|S_IWOTH, 1, getuid());
-									if (err != 0) {
-										snprintf(errbuf, sizeof(errbuf),
-											"error: %s file has a non-secure file access, errno: %d", filename, err);
-										error = 1;
-									}
+									#if !defined(DEBUG) && !defined(NO_SECURITY_CHECK)
```

Comment:
nit: I think #if should start from starting of line, also block inside is not aligned wrt outside of #if block (same for other file too)